tests/integration/s2n_handshake_test_s_client.py

```diff
@@ -102,7 +118,7 @@ def try_handshake(endpoint, port, cipher, ssl_version, server_cert=None, server_
     s2nd = subprocess.Popen(s2nd_cmd, stdin=subprocess.PIPE, stdout=subprocess.PIPE)
 
     # Make sure it's running
-    s2nd.stdout.readline()
+    sleep(0.1)
```

Comment:
Could this use poll() to check if the process is running? Or does s2nd need additional time to get into a state ready for input?

Reply:
Talked offline and it does need this extra time.